spec/ParseQuery.spec.js

```diff
@@ -980,7 +981,7 @@ describe('Parse.Query testing', () => {
       .then(done.fail)
       .catch(response => {
         equal(response.data.code, Parse.Error.INVALID_JSON);
-        equal(response.data.error, 'bad $containedBy: should be an array');
+        equal(response.data.error, ErrorMessage.queryValueTypeInvalid('an array', '$containedBy:'));
```

Comment:
This also looks odd, can we move the colon in `$containedBy:` into the error message?